### ISSUE TITLE
Not able to select date and time in WEB

### ISSUE THREAD

**Issue by TechnoPrashant** [koukibadr/Bottom-Picker#16]
- When I implemented it in the mobile app its work perfectly but on the web, it will open correctly but not able to select time and date.

**Comment by koukibadr** [comment]
@TechnoPrashant  thanks for sending me this bug I will work on it and let you know when a new version is available thanks for your support

**Comment by koukibadr** [comment]
@TechnoPrashant  can you please show me how you implemented the bottom picker in your app ? i'm not having this issue in the example that i've tested now

**Comment by TechnoPrashant** [comment]
@koukibadr  Thanks waiting for your update.

**Comment by TechnoPrashant** [comment]
@koukibadr Hi Any updates ?

**Comment by TechnoPrashant** [comment]
@koukibadr  It was fixed from my side. Issue from my side. But still, one thing I notice is that when I use a mouse to scroll picker it does not work.

**Comment by TahaMsv** [comment]
> @koukibadr It was fixed from my side. Issue from my side.

could you help us to fix this issue? please describe your solution.

**Comment by koukibadr** [comment]
@TahaMsv on the web the scroll works but with the middle mouse wheel not with dragging the list down
You can test it and let me know if it works or not

**Comment by TahaMsv** [comment]
> @TahaMsv on the web the scroll works but with the middle mouse wheel not with dragging the list down You can test it and let me know if it works or not

That's right, I knew. But I want to know how to work with the mouse down(drag). I opened an issue for this a few hours ago.

**Comment by TahaMsv** [comment]
> Okay for now it's not supported but I will add the support for dragging on the web version Thanks for your feedback @TahaMsv

Thanks. I need it for my project. Can you guess how long does it take to fix it?
